tests/codeception/wpunit/Pods/Field/PodsField_PickTest.php

```diff
@@ -89,6 +89,22 @@ public function test_display_format_multi_simple() {
 		$expected = 'item1 | item2 | item3';
 
 		$this->assertEquals( $expected, $this->field->display( $value, null, $options ) );
+
+		// List display format.
+
+		$options['pick_display_format_multi'] = 'ul';
+
+		$expected = '<ul><li>item1</li><li>item2</li><li>item3</li></ul>';
```

Comment:
We'll need to update this to match the new formatting if we update that in my prior suggestion

Reply:
Pending decision here. I personally wouldn't add `\n` in HTML formatting.